packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -148,9 +150,10 @@ export class EpochTracker implements IPersistedFileCache {
         const request = this.addEpochInRequest(url, fetchOptions, addInBody);
         let epochFromResponse: string | undefined;
         try {
-            const response = await this.rateLimiter.schedule(
-                async () => fetchAndParseAsJSONHelper<T>(request.url, request.fetchOptions),
-            );
+            const fetchFn = async () => fetchAndParseAsJSONHelper<T>(request.url, request.fetchOptions);
+            const response = fetchType === "createBlob"
```

Comment:
This feels like too implicit / hidden. I.e. should it be move out of epoch tracker and be higher level concept in OdspDocumentStorageService class?
I think it's Ok that it results on using both limiters in cascading fashion, as one for upload controls storage concurrency, the one in epoch controls not overhelping the browser with too many request (i.e. we likely still want to maintain 24 max for all requests combined, even if that makes blob upload requests lower in priority relative to others).